public/app/plugins/panel/graph/graph_tooltip.ts

```diff
@@ -49,11 +49,13 @@ export default function GraphTooltip(this: any, elem, dashboard, scope, getSerie
     }
   };
 
-  this.renderAndShow = (absoluteTime, innerHtml, pos, xMode) => {
+  this.renderAndShow = (absoluteTime, innerHtml, pos, xMode, primary) => {
     if (xMode === 'time') {
       innerHtml = '<div class="graph-tooltip-time">' + absoluteTime + '</div>' + innerHtml;
     }
     $tooltip.html(innerHtml).place_tt(pos.pageX + 20, pos.pageY);
+    // The primary tooltip should be drawn on top
+    $tooltip.css('zIndex', primary ? 9999 : 9998);
```

Comment:
what is not primary? can't see false ever being sent in

Reply:
It's due to the default value, like some of the other calls similar to the item argument, such as https://github.com/cylon/grafana/blob/master/public/app/plugins/panel/graph/graph.ts#L127.